exp/ingest/io/processors.go

```diff
@@ -0,0 +1,59 @@
+package io
+
+import (
+	"io"
+
+	"github.com/stellar/go/support/errors"
+)
+
+type ChangeProcessor interface {
+	ProcessChange(change Change) error
+}
+
+type LedgerTransactionProcessor interface {
+	ProcessTransaction(transaction LedgerTransaction) error
+}
+
+func StreamLedgerTransactions(
```

Comment:
I wonder if `Stream` explains what happens inside properly.
```suggestion
func ProcessLedgerTransactions(
```